types/sequelize/index.d.ts

```diff
@@ -5007,11 +5009,26 @@ declare namespace sequelize {
          */
         indicesType?: 'UNIQUE' | 'FULLTEXT' | 'SPATIAL';
 
+        /**
+         * The index type
+         */
+        type?: 'UNIQUE' | 'FULLTEXT' | 'SPATIAL';
```

Comment:
This is a bit of a nitpick, but it would be nice to have `indicesType` and `type` unified by a common type, i.e.:

```typescript
type IndicesType = 'UNIQUE' | 'FULLTEXT' | 'SPATIAL';

indicesType?: IndicesType;
type?: IndicesType;
```

Reply:
That's could not be bad.